src/bootstrap.c

```diff
@@ -83,6 +83,7 @@ int main(int argc, char **argv) {
     envar_set("SINGULARITY_DOCKER_USERNAME", singularity_registry_get("DOCKER_USERNAME"), 1);
     envar_set("SINGULARITY_CACHEDIR", singularity_registry_get("CACHEDIR"), 1);
     envar_set("SINGULARITY_version", singularity_registry_get("VERSION"), 1);
+    envar_set("SINGULARITY_ENVIRONMENT", "/.singularity.d/env/91-environment.sh", 1);
```

Comment:
OK.  I'll give it a shot.  Note that SINGULARITY_ENVIRONMENT is already a
(rather important) variable.  If I add it to `deffile-sections.sh` is it
going to get added to the registry?  What kind of havoc would that cause?